### PR TITLE
Code cleanup use Strings and fix Trailing Commas

### PR DESCRIPTION
**Pull request by workerbee22**
This is a code cleanup and consistency change. 

- See issue [197](https://github.com/Western-Health-Covid19-Collaboration/wh_covid19_app/issues/197) for the recommendation on the Strings class.
- trailing commas code formatting and alignment as per recommended approach [Flutter Trailing Commas](https://flutter.dev/docs/development/tools/formatting#using-trailing-commas)

## Changes

- String class use for on screen text, except for content/data
- `constants.dart` for app constants that should never change
- code reformatting to add trailing commas as per recommendation above

## Screenshots

No UI changes as this is a code cleanup exercise only. App tested and re run on Android and iOS to confirm no changes.

Note comparisons to Zeplin designs are yet to be done due to the time involved and the urgency on merging this during the day before other work on the code base starts later tonight. This change will make alignment to design easier.

## Things to note

Most files in the project have changed and so this is a very large PR. Please resist other changes in this PR, we can do it in others later.

Note there are still some future changes to `hard_data.dart` to use Strings more often that should be done, but plan to do that in another PR (given its just those two files involved) given the line between Strings and content/dart is unclear here.

### CODE REVIEW

**Review by domesticmouse — APPROVED** [review]

**Review by ericjiang97 — APPROVED** [review]

File: lib/hard_data.dart [hunk 1]
```diff
@@ -21,7 +28,7 @@ const List<ReusableCard> staffWelfare = [
     routeTo: Routes.ppe,
   ),
   ReusableCard(
-    title: 'Your Welfare',
+    title: Strings.yourWelfareTitle,
     description: 'Vital info & contacts',
```
Comment (rockgecko-development) [inline lib/hard_data.dart:32]: I wonder if it makes sense to extract the titles, but not the descriptions? I think it's fine with either both in strings, or both here.

**Comment by rockgecko-development** [comment]
LGTM :shipit:

File: lib/hard_data.dart (hunk 1 above)
Comment (workerbee22) [reply lib/hard_data.dart:32]: Yeah .... was up and down on that too. In `hard_data.dart` there are lots of these choices.

In this case the title is used multiple places in code so makes sense ie. 'Your Welfare' in the cared title on the Home screen, but also on the AppBar within that screen. So makes sense to keep those two in sync.

I left the description as only used once. So either would be fine I guess. But for example for ALL the strings in `hard_data.dart` I don't think we want to move them to Strings, as almost no advantage vs leave where they are. 

So deferred these choices for now. But at least its just these 2 files involved. So much easier to chnage this later.

**Review by rockgecko-development — APPROVED** [review]

**Comment by workerbee22** [comment]
OK with codemagic down, have done all local tests like `flutter analyze`  and `flutter test` and all passed. Also tested and found a bug, so a final commit to fix.

Now merging to master manually, skilling codemakic PR workflow build. So we can get this one done.

**Pull request merged**